panels/site_sections/attractions.py

```diff
@@ -19,6 +19,13 @@ def _attendee_for_badge_num(session, badge_num, options=None):
     return query.first()
 
 
+def _attendee_for_info(session, first_name, last_name, email, zip_code):
+    try:
+        return session.lookup_attendee(first_name, last_name, email, zip_code)
+    except:
+        return None
```

Comment:
As per @kitsuta 's suggestion I've updated this function to use ``session.lookup_attendee``.